### PR TITLE
Some small fixes to docs

### PR DESCRIPTION
**Pull request by micwoj92**

### CODE REVIEW

File: docs/os/windows.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Open your game folder through Steam. Then, right click on `hl2.exe`, click Prope
 
 More information: https://devblogs.microsoft.com/directx/demystifying-full-screen-optimizations/, https://reddit.com/r/Windows10/comments/645ukf/windows_10_cu_fullscreen_optimizations/dg330ub?context=3
 
-If disabling fullscreen optimizations helps, [let Microsoft know](https://aka.ms/fullscreenoptimizationsfeedback) so they can improve it in the future.
+If disabling fullscreen optimizations helps, [let Microsoft know](https://aka.ms/fullscreenoptimizationsfeedback), so they can improve it in the future.
```
[the remark this answers is not in the document]
Comment (micwoj92) [reply docs/os/windows.md:44]: lol

**Comment by micwoj92** [comment]
Maybe just delete link to that feedback page? What do you think @mastercoms ?

**Comment by mastercoms** [comment]
It's not dead. I closed the issue. It is just a different protocol that the link checker doesn't support.

**Comment by micwoj92** [comment]
I tried to open it and it didnt redirect, does it only do that on microsoft windows? https://aka.ms redirects me to microsoft website.

**Comment by mastercoms** [comment]
Yes, it's the feedback app protocol for MS windows 10 only.

File: docs/customization/modules.md [hunk 2]
```diff
@@ -333,7 +333,7 @@ Default setting: based on which preset you are currently using.
 
 * **`props=low`**: Disables client side props like bottles, disables foliage, low quality prop models and invisible railings.
 * **`props=medium`**: Disables client side props, disables foliage, default quality prop models.
-* **`props=high`**: Enables ambient lighting and decals on static props, enables a small number of client side props, enables foliage at a reasonable distance with instant pop in, default quality prop models.
+* **`props=high`**: Enables ambient lighting and decals on static props, enables some of client side props, enables foliage at a reasonable distance with instant pop in, default quality prop models.
```
Comment (mastercoms) [inline docs/customization/modules.md]: It's a small number, not some. It's an actual quantity.

**Review by mastercoms — CHANGES_REQUESTED** [review]

File: docs/customization/modules.md (hunk 2 above)
Comment (micwoj92) [reply docs/customization/modules.md]: Fine now?

**Review by mastercoms — APPROVED** [review]

**Pull request merged**